.prettierignore

```diff
@@ -0,0 +1,10 @@
+.github
+.astro
+node_modules
+public
+.gitignore
```

Comment:
We can format .gitignore and  .prettierignore

But should ignore dist.

Do we have any build or coverage folders? Or can we delete these entries?